### PR TITLE
Changes made to text file.

### PR DESCRIPTION
**Pull request by KevinSimbana**

### CODE REVIEW

File: GitGroupPractice.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@ Here is a file created by Cathy
 It doesn't have much in it yet.
 But it is a very nice file.
 Good morning all!
+
+
+Drake > Kanye West
```
Comment (Professional-Miles) [inline GitGroupPractice.txt:7]: Tragic.

**Review by Professional-Miles — CHANGES_REQUESTED** [review]
There's a discrepancy in this document.

**Review by bahmed98uk — APPROVED** [review]

**Review by bahmed98uk — APPROVED** [review]

**Review by Denis-Neagu — APPROVED** [review]
nice 1

**Review by bahmed98uk — APPROVED** [review]

**Comment by btara99** [comment]
good edit

**Review by bahmed98uk — APPROVED** [review]

**Review by Professional-Miles — APPROVED** [review]

**Pull request merged**